.github/workflows/workflow.yml

```diff
@@ -22,17 +21,20 @@ jobs:
       AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
       AWS_REGION: ${{ secrets.AWS_REGION }}
       EC2_SECRET: ${{ secrets.EC2_PWD }}
-      S3_BACKEND_KEY_NAME: tf-state-${{ github.sha }}
+      S3_BACKEND_KEY_NAME: TMP
     outputs:
-      runner: ${{ steps.determine-runner.outputs.runner }}
+      runner: ${{ env.S3_BACKEND_KEY_NAME }}
```

Comment:
"TMP" will be output, you have to specify which job you want to take output from

Reply:
The value is erased during the action

Reply:
`run: echo "S3_BACKEND_KEY_NAME=$(echo $(uuidgen))" >> $GITHUB_ENV`

Reply:
also this breaks behaviour in forks